app/src/main/java/de/westnordost/streetcomplete/quests/max_speed/AddMaxSpeed.kt

```diff
@@ -56,7 +56,31 @@ class AddMaxSpeed : OsmFilterQuestType<MaxSpeedAnswer>() {
                 changes.modify("highway", "living_street")
             }
             is ImplicitMaxSpeed -> {
+                if (answer.countryCode == "GB") {
+                    when(answer.roadType) {
+                        "nsl_restricted" -> {
+                            changes.add("maxspeed", "30 mph")
+                        }
+                        "nsl_single" -> {
+                            changes.add("maxspeed", "60 mph")
+                        }
+                        "nsl_dual" -> {
+                            changes.add("maxspeed", "70 mph")
+                        }
+                        "motorway" -> {
+                            changes.add("maxspeed", "70 mph")
+                        }
+                    }
+                }
```

Comment:
I don't want to have this in the app. I have reasoned about this in the past multiple times. In a nutshell:
- statuary speed limits could change anytime (see f.e. France a few years ago). It then would tag it wrong until an app update is made
- having a list of all default speed limits in the app for every country is unmaintainable. If it is added for one country, people will start asking why it can not also be added for their country etc.
- smart data consumers will need to have the ability to infer actual speed limits from `maxspeed:type` (etc) anyway, and there is a list for that in the wiki, even.

Reply:
That makes sense, I'll take it out